README.md

```diff
@@ -4,3 +4,10 @@
 
 - [Vagrant](https://www.vagrantup.com/docs/installation/)
 - [Ansible](http://docs.ansible.com/ansible/intro_installation.html#latest-releases-via-pip)
+
+## How to:
+
+```
```

Comment:
We should list the recommended versions of Ansible and Vagrant, probably with a link to the Vagrant bug issue.